sky/skylet/providers/aws/node_provider.py

```diff
@@ -11,7 +11,7 @@
 try:
     import ray._private.ray_constants as ray_constants
 except ImportError:
-    # SkyPilot: for local ray version lower than 2.0.1
+    # SkyPilot: for local ray version lower than 2.2.0
```

Comment:
Is this true? Thought this comment meant the first version of Ray where ray._private.ray_constants appeared.